zipline/data/minute_bars.py

```diff
@@ -550,9 +550,14 @@ def last_date_in_output_for_sid(self, sid):
             sizes = f.read()
         data = json.loads(sizes)
         num_days = data['shape'][0] / self._minutes_per_day
+
         if num_days == 0:
             # empty container
             return pd.NaT
+
+        # cast to int for pandas
+        # https://github.com/pandas-dev/pandas/blob/master/pandas/tseries/base.py#L247
+        num_days = int(num_days)
```

Comment:
I think that if we're going to convert to an int, we should be doing this earlier, before we even check for `num_days == 0`

We can use the `//` operator which does integer division in both Python 2 and 3, and do that on line 552:

`num_days = data['shape'][0] // self._minutes_per_day`